os/tools/mkbootparam.py

```diff
@@ -24,11 +24,17 @@
 bootparam_file_path = os.path.dirname(__file__) + '/../../build/output/bin/bootparam.bin'
 user_file_dir = os.path.dirname(__file__) + '/../../build/output/bin/user'
 
-SIZE_OF_BOOTPARAM_PART = 8192
+SIZE_OF_BOOTPARAM = 4096
+
+# The partition size of dual BPs
+SIZE_OF_BOOTPARAM_PART = SIZE_OF_BOOTPARAM * 2
```

Comment:
SIZE_OF_BOOTPARAM and SIZE_OF_BOOTPARAM_PART is difficult to distinguish what it is.
How about `SIZE_OF_BPx` and `SIZE_OF_ENTIRE_BP`?

Reply:
I updated `SIZE_OF_BPx` and `SIZE_OF_BP_PARTITION`